ui/component/router/index.js

```diff
@@ -1,17 +1,27 @@
 import { connect } from 'react-redux';
 import * as SETTINGS from 'constants/settings';
+import * as PAGES from 'constants/pages';
 import { selectUserVerifiedEmail } from 'redux/selectors/user';
 import { selectHasNavigated, selectScrollStartingPosition } from 'redux/selectors/app';
 import { selectClientSetting, selectHomepageData, selectWildWestDisabled } from 'redux/selectors/settings';
 import Router from './view';
-import { selectTitleForUri } from 'redux/selectors/claims';
+import { selectTitleForUri, selectClaimUriForId } from 'redux/selectors/claims';
 import { doSetHasNavigated, doSetActiveChannel } from 'redux/actions/app';
 import { doUserSetReferrer } from 'redux/actions/user';
 import { selectHasUnclaimedRefereeReward } from 'redux/selectors/rewards';
 import { selectUnseenNotificationCount } from 'redux/selectors/notifications';
+import { getPathForPage } from 'util/url';
+
+const PLAYLIST_PATH = getPathForPage(PAGES.PLAYLIST);
 
 const select = (state, props) => {
-  const { uri } = props;
+  let { uri } = props;
+  const { pathname } = state.router.location;
+
+  if (pathname.startsWith(PLAYLIST_PATH)) {
+    const collectionId = pathname.replace(PLAYLIST_PATH, '');
```

Comment:
--- Maybe ---
How are these playlist URLs shared?  Can't seem to find it in the share dialog .... the share dialog always includes the channel name.

Usually, a shared link will include a referrer ID, redirect param (like sunset, etc.).  I think this will not parse the `collectionId` correctly for these cases.

Reply:
This was intended for this page, ex: https://salt.odysee.com/$/playlist/73c2a30109229b2f262574cb7881f265f3284475 , vs the current: https://odysee.com/$/list/4df6cc75f73f220cc9b479d789b06702c0eede1e (page title is just "Odysee")

Reply:
Yup.  The concern is just whether the client, browser, or anything will add extra stuff after that. Some examples for other types of URLs where stuff is appended.
- https://odysee.com/?&sunset=lbrytv
- https://odysee.com/@SomeOrdinaryGamers:a?r=3vzdBMnXEhDiByz2x7R1RkCmUw1Hwk82

Reply:
Oh I see the replace is the problem now my bad

Reply:
just realized this wouldn't happen since it is using the value on `pathname` so it already strips other queries